components/PageHero/PageHero.vue

```diff
@@ -72,7 +72,7 @@ export default {
   }
   h1,
   p {
-    background-color: rgba(36, 36, 91, 0.75);
+    background-color: #383671;
```

Comment:
This can remain unchanged as the designs have it transparent so you can still somewhat see the image behind it, as pictured in this screen.

![staging sparc science_(iPad)](https://user-images.githubusercontent.com/1493195/82082817-a97d5900-96b6-11ea-99f6-7415efb5cfb0.png)

Reply:
@cameronbaney Ah, I see. I reverted it back to maintain the opacity, just changed the RGB values to reflect the new color.

Reply:
Excellent